File: Pyvvo.Logistics.Model/Model/Processing.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Pyvvo.Logistics.Model
{
    public class Processing
    {
        [Required, Key] public long Id { get; set; }
        [Required] public long StatusId { get; set; }
        [Required] public long OrderId { get; set; }
        [Required] public long CreatedById { get; set; }
        [Required] public long AgentId { get; set; }
        public long ReferenceNumberId { get; set; }
        public string ReferenceNumber { get; set; }
        [Required] public DateTime CreatedOn { get; set; }
        public DateTime UpdatedOn { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public bool IsActive { get; set; }
        public bool SendUpdate { get; set; }
        [Required] public Status Status { get; set; }
        [Required] public Order Order { get; set; }
        [Required] public User CreatedBy { get; set; }
        public User Agent { get; set; }
        public List<ProcessingLineItem> ProcessingLineItems { get; set; }
        public List<Note> Notes { get; set; }
        
    }
}

File: Pyvvo.Logistics/ClientApp/src/app/service/Model/processing.ts
import { Status } from "./status";
import { Order } from "./order";
import { User } from "./user";
import { ProcessingLineItem } from "./processing-line-item";
import { Note } from "./Note";

export class Processing {
  id: number;
  referenceNumber: string;
  referenceNumberId: number;
  createdOn: Date;
  updatedOn: Date;
  startDate: Date;
  endDate: Date;
  isActive: boolean;
  sendUpdate: boolean;
  status: Status;
  order: Order;
  createdBy: User;
  processingLineItems: ProcessingLineItem[];
  notes: Note[];
  agent: User;
}
